sklearn/kernel_approximation.py

```diff
@@ -670,6 +670,11 @@ class Nystroem(TransformerMixin, BaseEstimator):
         Pass an int for reproducible output across multiple function calls.
         See :term:`Glossary <random_state>`.
 
+    n_jobs : int, default=None
+        The number of jobs to use for the computation. This works by breaking
+        down the kernel matrix into n_jobs even slices and computing them in
+        parallel.
```

Comment:
Please add the directive: `.. versionadded:: 0.24` and add a short entry to the changelog (`doc/whats_new/0.24.rst`).

Reply:
added with the latest commit